documentation/SpectralTypes.md

```diff
@@ -0,0 +1,25 @@
+# SpectralTypes
+
+The SpectralTypes table contains spectral type measurements for sources listed in the Sources table. 
+The combination of *source*, *regime*, and *reference* is expected to be unique.
+Note that *regime* is a string constrained from a list of enumerated values.
+
+| Column Name | Description  | Unit  | Data Type | Key Type  |
+|---|---|---|---|---|
+| source    | Unique identifier for the source |   | String(100)  | primary and foreign: Sources.source   |
+| spectral_type | Spectral type string |  | String(10)  |   |
```

Comment:
Do we want to keep spectral type unconstrained like this or require e.g. first char must be of M,L,T,Y and second char inclusively betwixt 0-9? To stop things like "sdM9" or ">M10V"?

Reply:
please add this comment to my PR #94, which modifies the spectral type table.